### PR TITLE
chore: Use scalafmt to group imports

### PR DESCRIPTION
**Pull request by He-Pin**
Motivation:

Use scalafmt to group imports

### CODE REVIEW

**Review by natsukagami — COMMENTED** [review]
Otherwise lgtm, thanks so much! I thought I'd need scalafix to do this

**Comment by He-Pin** [comment]
BTW, I see there is no cleanInput method, but cleanResult, is that by design？

**Comment by natsukagami** [comment]
> cleanInput method, but cleanResult, 

Not sure which method you are mentioning here

File: .scalafmt.conf [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+    sort = ascii
+    groups = [
+      ["gears\\..*"],
+      ["scala\\.meta\\..*", "org\\.scalameta\\..*"],
+      ["sbt\\..*"],
+      ["java.?\\..*"],
+      ["scala\\..*"],
+      ["org\\..*"],
+      ["com\\..*"],
+    ]
```
Comment (natsukagami) [inline .scalafmt.conf:21]: Is it possible to group just by
1. Java & Scala  
2. gears internal imports
3. Everything else

? the current list feels a bit random

**Pull request merged**